server/routes/admin/users/userInfo.js

```diff
@@ -4,14 +4,25 @@ const knex = require("../../../utils/dbConfig");
 
 router.get("/:id", (req, res) => {
   knex("users")
+    .leftJoin(
+      "address",
+      "address.user_id",
+      "users.id"
+    )
     .select(
       "users.id",
       "users.username",
       "users.email",
       "users.first_name",
       "users.last_name",
       "users.is_admin",
```

Comment:
use "users.*" instead of writing all the field names
same for address also

Reply:
How to do that this was a select query

Reply:
that you have to figure out

Reply:
we can reduce a few lines if you do that right? why do we have to write all the field names if we want all the columns from that table?

Reply:
We are not selecting all the coloumns

Reply:
why not? admin side you can display all the columns right?

Reply:
is there any column that you don't want to show?

Reply:
please check now. although i am surprised to see that i selected from user table on the console log it shows data from address table as well, is it because of the join query that i placed before??